RealmSwift/Tests/ObjectCreationTests.swift

```diff
@@ -441,7 +441,7 @@ class ObjectCreationTests: TestCase {
         Realm().commitWrite()
         switch type {
             case .Bool:     return ["invalid", 2 as Int, 1.1 as Float, 11.1 as Double]
-            case .Int:      return ["invalid", true, false, 1.1 as Float, 11.1 as Double]
+            case .Int:      return ["invalid", 1.1 as Float, 11.1 as Double]
```

Comment:
In obj-c assigning a boolean value to an int property is perfectly legal and sensible, and NSNumber started actually returning a boolean type from objcType when given boolean boxed in obj-c.